sim/src/main.rs

```diff
@@ -50,40 +50,73 @@ fn main() {
     }
 
     // Create simulator object.
-    let mut simulator: Simulator = Simulator::new(seed);
+    let storage_name = "storage";
+    let mut simulator = sim::simulator::Simulator::new(1); // always run with the seed 0 while we are checking tests
 
-    // node arguments go:  id, capacity, egress_rate, generation_rate, plugin
-    simulator.add_node("traffic-gen", 1, 1, 1, plugin_str);
-    simulator.add_node("node-1", 10, 1, 0, plugin_str);
-    simulator.add_node("node-2", 10, 1, 0, plugin_str);
-    simulator.add_node("node-3", 10, 1, 0, plugin_str);
-    simulator.add_node("node-4", 1, 1, 0, plugin_str); // in setting egress rate to 0, we are making a sink
-    simulator.add_node("node-5", 1, 1, 0, plugin_str); // in setting egress rate to 0, we are making a sink
-    simulator.add_storage("storage");
+    let regular_nodes = [
+        "frontend-v1",
+        "cartservice-v1",
+        "productcatalogservice-v1",
+        "currencyservice-v1",
+        "paymentservice-v1",
+        "shippingservice-v1",
+        "emailservice-v1",
+        "checkoutservice-v1",
+        "recomendationservice-v1",
+        "adservice-v1",
+    ]
+    .to_vec();
```

Comment:
Why are we using hipster shop instead of bookinfo? Just for this example? In tracing_env we should use the bookinfo though.

Reply:
I had seen some of the example queries use terms from hipster shop;  now that we have realized the example queries were messed up anyway, the choice has no reason behind it.  I changed it back.